helm-charts/basehub/templates/configmap-hub-templates.yaml

```diff
@@ -0,0 +1,11 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: hub-custom-templates-config
+  labels:
+    app: jupyterhub
+data:
+  {{- with .Values.jupyterhub.custom.homepage }}
+  HUB_TEMPLATES_REPO_URL: {{ .gitRepoUrl | quote }}
+  HUB_TEMPLATES_REPO_BRANCH: {{ .gitRepoBranch | quote }}
+  {{- end }}
```

Comment:
Maybe rename to GIT_REPO_X now for consistency between naming? A bit nitty comment but I figure its a bit helpful to stay consistent for quick code comprehension